### PR TITLE
Fix update omim

### LINKED ISSUE #1544: "scout update omim" not working
Worked a few weeks ago... I don't have time to debug this at the moment. If no one else cares about this I will take a look at it next week!

```
$ scout update omim --institute CMD --api-key xxxxxxxxxxxxxxxxxxxxx

2019-11-21 15:17:13 cmdscout2 scout.commands.base[16298] INFO Running scout version 4.8.3
2019-11-21 15:17:13 cmdscout2 scout.commands.base[16298] DEBUG Debug logging enabled.
2019-11-21 15:17:13 cmdscout2 scout.commands.update.omim[16298] INFO Running scout update omim
2019-11-21 15:17:13 cmdscout2 scout.adapter.mongo.institute[16298] DEBUG Fetch institute CMD
2019-11-21 15:17:13 cmdscout2 scout.adapter.mongo.panel[16298] INFO Fetching gene panels OMIM-AUTO from database
2019-11-21 15:17:13 cmdscout2 scout.adapter.mongo.panel[16298] INFO Setting version to 3.0
2019-11-21 15:17:13 cmdscout2 scout.utils.requests[16298] INFO Fetching OMIM files from https://omim.org/
2019-11-21 15:17:13 cmdscout2 scout.utils.requests[16298] INFO Requesting https://omim.org/static/omim/data/mim2gene.txt
2019-11-21 15:17:14 cmdscout2 scout.utils.requests[16298] INFO Requesting https://data.omim.org/downloads/xxxxxxxxxxxxxxxxx/genemap2.txt
2019-11-21 15:17:15 cmdscout2 scout.adapter.mongo.hgnc[16298] INFO Fetching all genes by alias
2019-11-21 15:17:16 cmdscout2 scout.parse.omim[16298] INFO Get the mim genes
2019-11-21 15:17:16 cmdscout2 scout.parse.omim[16298] INFO Parsing mim2gene
2019-11-21 15:17:16 cmdscout2 scout.parse.omim[16298] INFO Number of genes without hgnc symbol 175
2019-11-21 15:17:16 cmdscout2 scout.parse.omim[16298] INFO Parsing the omim genemap2
2019-11-21 15:17:16 cmdscout2 scout.commands.update.omim[16298] ERROR 'Mim Number'
Aborted!
```

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #1544. 
This should be already working but I'm trying a test with real data-like omim info. Let's see if I manage

**How to test**:
1. On a local implementation of Scout run the following command:
`scout --demo update omim --institute cust000 --api-key a_valid_api_key`

**Expected outcome**:
The command should run until the end and not return error

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
I think I found the change (MIM) but would it be possible to do this without the whitespace changes? 😊

**Comment by northwestwitch** [comment]
> be

They are super-annoying right?

**Comment by northwestwitch** [comment]
> I think I found the change (MIM) but would it be possible to do this without the whitespace changes? 😊

Yes it's only the MIM. The spaces are soooo annoying, right?

**Comment by northwestwitch** [comment]
Now Travis is passing and looks like everything is in place. Tried to revert back to the file without my formatting and apply the MIM change and push it back but the problem is that my editor doesn't really see the difference with master so it didn't work. :(

**Comment by northwestwitch** [comment]
Ready for review as soon as it passes the tests

**Comment by dnil** [comment]
Your editor is way too clever - almost like MS Word.. 😉

File: scout/parse/omim.py [hunk 1]
```diff
@@ -83,12 +83,12 @@ def parse_genemap2(lines):
             continue
 
         parsed_entry = parse_omim_line(line, header)
-        parsed_entry['mim_number'] = int(parsed_entry['Mim Number'])
+        parsed_entry['mim_number'] = int(parsed_entry['MIM Number'])
```
Comment (dnil) [inline scout/parse/omim.py]: :+1:

File: tests/parse/test_parse_omim.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "# Generated: 2017-02-02\n",
     "# See end of file for additional documentation on specific fields\n",
     "# Chromosome\tGenomic Position Start\tGenomic Position End\tCyto"\
-    " Location\tComputed Cyto Location\tMim Number\tGene Symbols\tGene Name"\
+    " Location\tComputed Cyto Location\tMIM Number\tGene Symbols\tGene Name"\
```
Comment (dnil) [inline tests/parse/test_parse_omim.py]: ✅

File: scout/demo/resources/genemap2_reduced.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 1966-2018 Johns Hopkins University. Use of this file adheres to the terms specified at https://omim.org/help/agreement.
 # Generated: 2018-04-24
 # See end of file for additional documentation on specific fields
-# Chromosome	Genomic Position Start	Genomic Position End	Cyto Location	Computed Cyto Location	Mim Number	Gene Symbols	Gene Name	Approved Symbol	Entrez Gene ID	Ensembl Gene ID	Comments	Phenotypes	Mouse Gene Symbol/ID
+# Chromosome	Genomic Position Start	Genomic Position End	Cyto Location	Computed Cyto Location	MIM Number	Gene Symbols	Gene Name	Approved Symbol	Entrez Gene ID	Ensembl Gene ID	Comments	Phenotypes	Mouse Gene Symbol/ID
```
Comment (dnil) [inline scout/demo/resources/genemap2_reduced.txt]: ✅

Comment (northwestwitch) [reply scout/demo/resources/genemap2_reduced.txt]: Thanks!

Comment (dnil) [reply scout/demo/resources/genemap2_reduced.txt]: You did test it, right? 😊

Comment (northwestwitch) [reply scout/demo/resources/genemap2_reduced.txt]: I did! If you want to give it a try I can give you @moonso's key! :D

**Comment by dnil** [comment]
I have it, but am to cheap to give him 10kr.

**Comment by bjhall** [comment]
Thanks for fixing this @northwestwitch ! Works fine for us as well. Such an annoying and unnecessary change on OMIM's side...

**Pull request merged**